src/graph/planner/ngql/FetchEdgesPlanner.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef GRAPH_PLANNER_NGQL_FETCH_EDGES_H_
```

Comment:
pragma once

Reply:
In my opinion, the effect is the same

Reply:
It's the new code style according to the doc.